CoreWhatsNewView/src/au/gov/asd/tac/constellation/views/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2021-05-19 Added the ability to pin the position of vertexes to prevent them from moving when using arrangement plugins 
```

Comment:
I would change the title to something like "Pinned Nodes" or "Ability to Pin/Unpin Nodes" to keep it short and simple